addons/l10n_in/views/report_invoice.xml

```diff
@@ -42,11 +42,13 @@
         <xpath expr="//h2" position="replace">
             <h2>
                 <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'" t-field="o.journal_id.name"/>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'draft'">Draft <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_refund'">Credit Note</span>
-                <span t-if="o.move_type == 'in_refund'">Vendor Credit Note</span>
-                <span t-if="o.move_type == 'in_invoice'">Vendor Bill</span>
+                <span t-elif="o.move_type == 'out_invoice' and o.state == 'draft'">Draft <span t-field="o.journal_id.name"/></span>
+                <span t-elif="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled <span t-field="o.journal_id.name"/></span>
+                <span t-elif="o.move_type == 'out_refund' and o.state == 'posted'">Credit Note</span>
+                <span t-elif="o.move_type == 'out_refund' and o.state == 'draft'">Draft Credit Note</span>
+                <span t-elif="o.move_type == 'out_refund' and o.state == 'cancel'">Cancelled Credit Note</span>
+                <span t-elif="o.move_type == 'in_refund'">Vendor Credit Note</span>
+                <span t-elif="o.move_type == 'in_invoice'">Vendor Bill</span>
                 <span t-field="o.name"/>
```

Comment:
```suggestion
                <span t-if="o.name != '/'" t-field="o.name"/>
```
Shouldn't there be a condition for `o.name` like in the other instances ?

Reply:
Good idea. I changed it according to the generic one.